### PR TITLE
add tfsec

### PR DESCRIPTION
**Pull request by edolsen-rv**

### CODE REVIEW

File: .github/workflows/tfsec.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          cli_config_credentials_token: ${{ secrets.tf_api_token }}
+      - name: Terraform Init
+        run: terraform init
+      - name: Terraform security scan
+        uses: triat/terraform-security-scan@v3.0.3
+        with:
+          tfsec_exclude: "AWS001,AWS002,AWS003,AWS004,AWS005,AWS006,AWS007,AWS008,AWS009,AWS010,AWS011,AWS012,AWS013,AWS014,AWS015,AWS016,AWS017,AWS018,AWS019,AWS020,AWS021,AWS022,AWS023,AWS024,AWS025"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```
Comment (RobvH) [inline .github/workflows/tfsec.yml]: badoink 
image

**Review by RobvH — CHANGES_REQUESTED** [review]

File: .github/workflows/tfsec.yml (hunk 1 above)
Comment (RobvH) [reply .github/workflows/tfsec.yml]: Please make this setting :) 
image

File: .github/workflows/tfsec.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          cli_config_credentials_token: ${{ secrets.tf_api_token }}
+      - name: Terraform Init
+        run: terraform init
+      - name: Terraform security scan
+        uses: triat/terraform-security-scan@v3.0.3
+        with:
+          tfsec_exclude: "AWS001,AWS002,AWS003,AWS004,AWS005,AWS006,AWS007,AWS008,AWS009,AWS010,AWS011,AWS012,AWS013,AWS014,AWS015,AWS016,AWS017,AWS018,AWS019,AWS020,AWS021,AWS022,AWS023,AWS024,AWS025"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (RobvH) [inline .github/workflows/tfsec.yml:23]: This GitHub action needs a GitHub token?

File: .tfsec/_tfchecks.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
+checks:
+- code: CU001
+  description: Custom check to ensure the CostCentre tag is applied to EC2 instances
+  impact: By not having CostCentre we can't keep track of billing
+  resolution: Add the CostCentre tag
```
Comment (RobvH) [inline .tfsec/_tfchecks.yml]: Did you intend to enforce "CostCentre" or is this an initial check-in?

Comment (edolsen-rv) [reply .tfsec/_tfchecks.yml]: This is an initial checkin to show a failure mode.
Will be switched to our normal tags after that.

File: .github/workflows/tfsec.yml (hunk 1 above)
Comment (edolsen-rv) [reply .github/workflows/tfsec.yml]: Double checked to make sure it was still enabled. Not sure why the newline is missing in this file.

File: .github/workflows/tfsec.yml (hunk 2 above)
Comment (edolsen-rv) [reply .github/workflows/tfsec.yml:23]: This is part of the examples shown by the author here: https://github.com/triat/terraform-security-scan/tree/master and in the live action here: https://github.com/triat/terraform-security-scan/blob/master/.github/workflows/MasterCI.yml

File: .github/workflows/tfsec.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        uses: hashicorp/setup-terraform@v2
+        with:
+          cli_config_credentials_token: ${{ secrets.tf_api_token }}
+      - name: Terraform Init
+        run: terraform init
+      - name: Terraform security scan
+        uses: triat/terraform-security-scan@v3.0.3
+        with:
+          tfsec_exclude: "AWS001,AWS002,AWS003,AWS004,AWS005,AWS006,AWS007,AWS008,AWS009,AWS010,AWS011,AWS012,AWS013,AWS014,AWS015,AWS016,AWS017,AWS018,AWS019,AWS020,AWS021,AWS022,AWS023,AWS024,AWS025"
+
```
Comment (RobvH) [inline .github/workflows/tfsec.yml]: lol still there: 
image

**Review by RobvH — DISMISSED** [review]

**Review by RobvH — DISMISSED** [review]

**Comment by zdiaz2826** [comment]
Closing PR because TFSec does not need to be enabled on TF modules.

**Pull request closed without merging**